content/en/contribute/code/core/update-dependencies.md

```diff
@@ -51,12 +52,44 @@ Then for each folder go through these steps.
   ```
 
 - Make sure the version of `api/enketo-xslt` is the same as `webapp/enketo-core/enketo-transformer/enketo-xslt`.
-- Don't update `lodash/core` since 4.17.21 is failing, this is an [open issue](https://github.com/lodash/lodash/issues/4904) in their repository.
-- Don't update `helmet` since it is not supporting NodeJS 8 anymore. Its minimum is NodeJS 10+.
-- If you have trouble upgrading any other dependency and you think it'll be challenging to fix it then raise a new issue to upgrade just that dependency. Don't hold up all the other upgrades you've made.
+- Don't update `lodash` to version `4.17.21`, there's a runtime issue when importing `lodash/core` in Admin app:
```

Comment:
Don't update `lodash` to version `4.17.21`. If you do so, the runtime issue highlighted below will appear when importing `lodash/core` in Admin app: